packages/mongodb-memory-server-core/package.json

```diff
@@ -54,6 +54,7 @@
     "get-port": "^5.1.1",
     "https-proxy-agent": "^5.0.0",
     "lockfile": "^1.0.4",
+    "lookpath": "^1.1.0",
```

Comment:
added this to check if a command is present. I think its handy, but of course there are more methods to check that.